deployments/dockerfiles/gosec/Dockerfile

```diff
@@ -1,23 +1,8 @@
 # Dockerfile used to create "husyci/gosec" image
 # https://hub.docker.com/r/huskyci/gosec/
 
-FROM golang:alpine as builder
+FROM securego/gosec:latest
 
-RUN apk update && apk upgrade \
-	&& apk add --no-cache alpine-sdk git \
-	&& apk add --update --no-cache git openssh-client jq
+RUN apk update && apk upgrade && apk --no-cache add ca-certificates openssh-client jq
```

Comment:
I think `RUN apk update && apk upgrade`  is a bad idea, as the result is non-deterministic. The Upstream image `securego/gosec:latest` should take care of keeping the base up to date.

Reply:
The `ca-certificates` package is installed in our base image. The second `apk add ca-certificates` should be removed

Reply:
Git should also be included in the upstream image.

Reply:
I'm removing the `apk update && upgrade`, but are you sure that installing the ca-certificates on huskyCI's base image will make them available inside gosec's container?
I think it will work only if there is a mount volume with the ca-certificates path set on bootstrap. WDYT?

Reply:
> but are you sure that installing the ca-certificates on huskyCI's base image will make them available inside gosec's container?

I meant it the other way around. If `ca-certificates` is installed in `securego/gosec`, our image should inherited these files